deployment/morph/webghc.nix

```diff
@@ -33,7 +33,7 @@
 
       networking.firewall = {
         enable = true;
-        allowedTCPPorts = [ 80 9100 9091 9113 ];
+        allowedTCPPorts = [ 22 80 9100 9091 9113 ];
```

Comment:
I'm seeing `allowedTCPPorts` in several places, i wonder if it would be feasible to extract/import it? It's probably fine either way speaking of myself i certainly remember banging my head against the wall because of some screw-up with conflicting port specifications..

Just a thought. Maybe anything "smarter" than you have now just makes it more complicated. I leave that up to you

Reply:
ok I've added a `ports` object to configuration.nix which I now thread through. This should avoid any problems using incorrect ports in the future.